src/recipes/index.md

```diff
@@ -0,0 +1,46 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Recipes
+
+<!-- Using a lower heading to allow expanding with more categories of recipes. Not sure what an h1 heading would contain exactly -QoS -->
+## Tooling
```

Comment:
As stated, "Tooling" is not the best terminology. Please remove it from the whole page.

You can keep things flat for now (everything directly under `Recipes`). It's always possible to add more hierarchy once patterns emerge, but separation between "Tooling" and "Misc" seems somewhat arbitrary.

In general, we should consider if we want to keep this intro page once the pages grow; as it needs to be maintained alongside. But for now it's fine 🙂